### PR TITLE
Modify autoscale's printer

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/autoscaling/types.go#L80
If we don's set MinReplicas, its default value is 1.
Not \<unset\>
image

So I think replace <unset> with 1 is preferred.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL

File: pkg/printers/internalversion/printers.go [hunk 1]
```diff
@@ -1579,10 +1579,10 @@ func printHorizontalPodAutoscaler(obj *autoscaling.HorizontalPodAutoscaler, opti
 	reference := fmt.Sprintf("%s/%s",
 		obj.Spec.ScaleTargetRef.Kind,
 		obj.Spec.ScaleTargetRef.Name)
-	minPods := "<unset>"
+	minPods := int32(1)
```
Comment (smarterclayton) [inline pkg/printers/internalversion/printers.go]: The server is responsible for handling defaulting.  We cannot interpret it on the client.

Comment (zhangxiaoyu-zidif) [reply pkg/printers/internalversion/printers.go]: @smarterclayton  thanks, I will close this PR. =)

**Pull request closed without merging**